### PR TITLE
Improve level1.js and fix bug in level3.js

### PR DESCRIPTION
**Pull request by danny-wu**
Just some suggested changes from what I've observed mentoring this workshop, and a bugfix. See inline comments :)

### CODE REVIEW

File: js/level1.js [hunk 1]
```diff
@@ -4,34 +4,35 @@
...
 
     They will help us to guide you through the JavaScript introduction
     journey.
 */
 
+// Single line comments look like this.
+
 /*
     Multi-line comments look like this.
+    When you are writing actual code, put it outside these comment blocks,
```
Comment (danny-wu) [inline js/level1.js]: People often get confused and put code inside the comments. Hopefully this makes it clearer

File: js/level1.js [hunk 2]
```diff
@@ -4,34 +4,35 @@
...
-// Single line comments look like this.
-
-
 /*
     Let's start with getting your code on the screen.
 
     There are a few ways you can do it and we will look into a few of them:
 
-    * alert('Hello girls!');
+    alert('Hello girls!');
```
Comment (danny-wu) [inline js/level1.js]: People usually write things with the *, like `*alert('Hello girls!');`.

`*` is not used anywhere else in these files; so hopefully this makes it clearer.

File: js/level1.js [hunk 3]
```diff
@@ -123,8 +125,7 @@
     alert(sumOfNumbers);
 
     This will pop-up an alert box with the number 4.
-
-    (You can see a full list of )
+    TIP: Note how we didn't put 1 and 3 in quotes, because they are numbers.
```
Comment (danny-wu) [inline js/level1.js]: clarify, so people don't do `'1' + '3'` like people often do

File: js/level3.js [hunk 4]
```diff
@@ -261,7 +261,7 @@
 
     Example:
 
-    var pageNode = document.querySelector('body')[0];
```
Comment (danny-wu) [inline js/level3.js]: Accessing `[0]` on a single Element is going to give you an error. You only need to do `[0]` if you're using `document.querySelectorAll`

**Comment by danny-wu** [comment]
cc @ButenkoT

File: js/level1.js [hunk 5]
```diff
@@ -143,9 +144,9 @@
     Functions
     =========
 
-    A function is a reusable piece of code, an action that you want to do.
-    It takes some input (arguments), does some manipulation on it and returns
-    the output. Use the keyword 'return' to define the return value.
+    A function is lkike a blueprint, an action that you want to do.
```
Comment (ButenkoT) [inline js/level1.js]: mistype here in `lkike`

**Pull request merged**